Library/Homebrew/utils/gems.rb

```diff
@@ -144,7 +144,7 @@ def install_bundler_gems!(only_warn_on_failure: false, setup_path: true)
 
       # for some reason sometimes the exit code lies so check the output too.
       if bundle_check_failed || bundle_check_output.include?("Install missing gems")
-        unless system bundle, "install"
+        unless system_redirect_stdout_to_stderr bundle, "install"
           message = <<~EOS
             failed to run `#{bundle} install`!
           EOS
```

Comment:
I erred on the side of making the minimal changes here, but I believe if we changed this to:

```suggestion
        begin
          safe_system_redirect_stdout_to_stderr bundle, "install"
        rescue ErrorDuringExecution =>
          message = <<~EOS
            failed to run `#{bundle} install`!
          EOS
```

then we could get rid of the `system_redirect_stdout_to_stderr` and just use the `safe_` one everywhere. Not sure if this is preferable.

Reply:
Did this ^